site/introduction.md

```diff
@@ -6,67 +6,67 @@ head:
       content: Introduction
   - - meta
     - name: description
-      content: Learn what Account Kit is, how it works, and how web3 application developers can use it to add support for smart accounts.
+      content: Learn what Account Kit is, how it works, and how you can use it to add support for smart accounts.
   - - meta
     - property: og:description
-      content: Learn what Account Kit is, how it works, and how web3 application developers can use it to add support for smart accounts.
+      content: Learn what Account Kit is, how it works, and how you can use it to add support for smart accounts.
   - - meta
     - name: twitter:title
       content: Introduction
   - - meta
     - name: twitter:description
-      content: Learn what Account Kit is, how it works, and how web3 application developers can use it to add support for smart accounts.
+      content: Learn what Account Kit is, how it works, and how you can use it to add support for smart accounts.
 ---
 
 # What is Account Kit?
 
 **Account Kit** is a framework to embed smart accounts in your web3 app, unlocking [powerful features](/getting-started) like email/social login, gas sponsorship, batched transactions, and more. The `aa-sdk` makes it easy to integrate and deploy smart accounts, send user operations, and sponsor gas with just a few lines of code.
 
-## What is included in the Account Kit stack?
+## What is included in Account Kit stack?
 
 Account Kit is a complete solution for [account abstraction](https://www.alchemy.com/overviews/what-is-account-abstraction). It includes five components:
 
-- **AA-SDK**: A simple, powerful interface to integrate, deploy, and use smart accounts. The `aa-sdk` orchestrates everything under the hood to make development easy.
-- **LightAccount:** Secure, audited smart contract accounts. Easy to deploy, just when your users need them.
-- **Signer:** Integrations with the most popular wallet providers. Secure your accounts with email, social login, passkeys, or a self-custodial wallet signer.
+- **aa-sdk**: A simple, powerful interface to integrate, deploy, and use smart accounts. The `aa-sdk` orchestrates everything under the hood to make development easy.
+- **Light Account:** Secure, audited smart accounts. Easy to deploy, just when your users need them.
+- **signer:** Integrations with the most popular wallet providers. Secure your accounts with email, social login, passkeys, or a self-custodial wallet signer.
```

Comment:
see comment on capitalizing Signer since it's a new definition, similar to Provider and Account

Reply:
Capitalized!